### PR TITLE
Create about.js (Translated to spanish)

### LINKED ISSUE #47: translate app into español / traducir aplicación en español
### Expected Behavior

Spanish speakers can use the application en español.

### Task

All translations are located in https://github.com/mirai-audio/mir/tree/master/app/locales, with directory for each translation language. 

These should be added under a new `es` directory in `app/locales` according to the `ISO 639-1` standard names.

Please use English as the source for translations, as it contains a complete list of up-to-date strings.

Please use our [Writing Styleguide](https://github.com/mirai-audio/mir/wiki/Writing-Styleguide) for how the voice & persona of the text should come across.

### PR DESCRIPTION
**Pull request by andresluna11**
about.js translated to spanish.

# Goal / Issue

_Describe a short description of of this PR._

Resolves #47 

### Quality

* [ ] Submitted a ticket for my issue if one did not already exist
* [ ] Ticket number is in the subject of the commit message
* [ ] Used [Github auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message or PR
* [ ] Updated documentation for changes to the API
* [ ] Added new tests for new functionality, and tests are passing
* [ ] Code, writing, and design changes follow the [Style guides](https://github.com/mirai-audio/mir/wiki/Style-Guides)
* [ ] Single commit - `git rebase -i` off `master`
* [ ] No dangling orphan code

_If there’s a checkbox you can’t complete for any reason, that's okay, just
explain in detail why you weren’t able to do so._

### CODE REVIEW

**Comment by andresluna11** [comment]
I am going to translate all the files but I dont know how to sort them into a folder called "es".

**Comment by 0xadada** [comment]
No need to sort them, they just go into the same folder https://github.com/andresluna11/mir/tree/63f23d06efd6c61b7b16b838cf4d3193af1212ee/app/locales/es
🎉

File: app/locales/es/about.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable quotes, quote-props */
+
+export default {
+  'copy': 'La app de música para unir todos los servicios de música en streaming.  Poner canciones ' +
+    'De Sporify, Last.fm, Soundcloud, YouTube, mp3s. Crea listas de reproducción ' +
```
Comment (0xadada) [inline app/locales/es/about.js]: "Spotify"

**Comment by andresluna11** [comment]
Okey, I will do it today, But later, now Im studying.

**Comment by 0xadada** [comment]
Someone else has contributed a Spain-spanish "es-ES" translation here

https://github.com/mirai-audio/mir/pull/53

I'm going to close this PR, unless your translation is a different country. Which country are you translating for?

**Pull request closed without merging**